Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/plate.yml

```diff
@@ -87,6 +92,11 @@
             max: 1
       - !type:DoActsBehavior
         acts: [ "Destruction" ]
+  - type: Tag
+    tags:
+    - Recyclable
+    - Trash
+  - type: Recyclable
```

Comment:
FoodPlate already has the tag from above.

Reply:
What do you mean "from above?" Do you mean that FoodPlateSmall inherits the tag from 'parent: FoodPlate'? Let me double check in testing that the tag inherits correctly.

Reply:
Updated PR to remove the inherited tag.